packages/react-core/src/components/OverflowMenu/examples/OverflowMenu.md

```diff
@@ -82,15 +86,25 @@ class SimpleOverflowMenu extends React.Component {
           </OverflowMenuGroup>
         </OverflowMenuContent>
         <OverflowMenuControl>
-          <Dropdown
+          <DropdownNext
```

Comment:
One other nit: for the examples, it would be worth passing the `onOpenChange` callback since we're using DropdownNext, that way the menu can be closed by Escape/outside click. Alternatively, would we want to revisit whether that should be built into the component by default, and onOpenChange just being a way to provide customized handling?